core/controllers/features.py

```diff
@@ -51,11 +52,10 @@ def get(self, exploration_id: str) -> None:
         Args:
             exploration_id: str. The ID of the exploration.
         """
-        whitelisted_exploration_ids_for_playthroughs = (
-            config_domain.WHITELISTED_EXPLORATION_IDS_FOR_PLAYTHROUGHS.value)
         self.render_json({
-            'is_exploration_whitelisted':
-                exploration_id in whitelisted_exploration_ids_for_playthroughs,
+            'is_exploration_curated':
```

Comment:
Prefer exploration_is_curated (for booleans)

Reply:
Updated.